dev-client/src/services.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+export const getElevation = async (lat: number, lng: number) => {
+  // TypeScript complains if the values passed to URLSearchParams are floats instead of strings.
```

Comment:
nit: the reason TypeScript complains is because `URLSearchParams` requires its arguments to be strings, so it'd be simpler/more accurate to just say "URLSearchParams requires its arguments to be strings"